resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Constants.scala

```diff
@@ -45,6 +47,7 @@ private[spark] object Constants {
   // Default and fixed ports
   val DEFAULT_DRIVER_PORT = 7078
   val DEFAULT_BLOCKMANAGER_PORT = 7079
+  val DEFAULT_UI_PORT = UI_PORT.defaultValue.get
```

Comment:
I actually meant not define this constant at all, since it's already defined.

Reply:
I have removed this constant and pushed the correct changes.